packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -0,0 +1,60 @@
+Name:     foreman-release
+Version:  1.19.0
+Release:  1%{?dist}
+
+Summary:  Foreman repositories meta-package
+Group:    Applications/System
+License:  GPLv3+
+URL:      http://theforeman.org
+Source0:  %{name}.repo
+Source1:  %{name}-plugins.repo
+Source2:  %{name}.gpg
+
+BuildArch: noarch
+
+BuildRequires: sed
+
+%description
+Foreman repository contains open source and other distributable software for
+distributions in RPM format. This package contains the repository configuration
+for Yum.
+
+%package -n foreman-release-plugins
```

Comment:
I think you can use `%package plugins` to automatically name it `foreman-release-plugins`